Sources/StreamVideoSwiftUI/CallingViews/MicrophoneChecker.swift

```diff
@@ -87,12 +87,13 @@ public class MicrophoneChecker: ObservableObject {
                 guard let self = self else { return }
                 audioRecorder.updateMeters()
                 let decibel = audioRecorder.averagePower(forChannel: 0)
-                var temp = self.decibels
-                temp.append(decibel)
+                let normalisedAudioLevel = self.audioNormaliser.normalise(decibel)
```

Comment:
why is this `do {}` block so big? isn't better to limit to the only part of the code that can raise?

Reply:
I believe it was to make use of the non-optional instance of audioRecorder without the need for a name different than `audioRecorder`. I have now updated it.